docs/platforms/javascript/guides/nuxt/install/dynamic-import.mdx

```diff
@@ -67,7 +47,33 @@ As a temporary workaround, you can add the following overrides in your applicati
 }
 ```
 
-You can also check out the guide for installing the SDK with a <PlatformLink to="/install/cli-import">CLI flag `--import`</PlatformLink> or a <PlatformLink to="/install/top-level-import/">top-level import</PlatformLink>.
+You can also check out the guide for installing the SDK with the <PlatformLink to="/install/cli-import">CLI flag `--import`</PlatformLink> or <PlatformLink to="/install/limited-server-tracing">limited-server-tracing</PlatformLink>.
+
+## Initializing Sentry with Dynamic `import()`
+
+Enable the dynamic `import()` by setting `autoInjectServerSentry`:
+
+```typescript {filename:nuxt.config.ts} {3}
+export default defineNuxtConfig({
+  sentry: {
+    autoInjectServerSentry: 'experimental_dynamic-import'
+  },
+})
+```
+
+After setting this, the Sentry Nuxt SDK adds some build-time configuration to ensure your app is wrapped with `import()`.
+With this, Sentry can be initialized before all other modules of the application.
```

Comment:
```suggestion
After setting this, the Sentry Nuxt SDK will add build-time configuration so that your app will be wrapped with `import()`, ensuring that Sentry can be initialized before any other modules.
```